crates/sncast/tests/e2e/script/init.rs

```diff
@@ -0,0 +1,135 @@
+use camino::Utf8PathBuf;
+use indoc::{formatdoc, indoc};
+use snapbox::cmd::{cargo_bin, Command};
+use sncast::helpers::constants::SCRIPTS_DIR;
+use sncast::helpers::scarb_utils::get_cairo_version;
+use tempfile::TempDir;
+
+#[test]
+fn test_script_init_files_created() {
+    let script_name = "my_script";
+    let temp_dir = TempDir::new().expect("Unable to create a temporary directory");
+
+    let snapbox = Command::new(cargo_bin!("sncast"))
+        .current_dir(temp_dir.path())
+        .args(["script", "init", script_name]);
+
+    snapbox.assert().stdout_matches(formatdoc! {r"
+        command: script init
+        status: Successfully initialized `{script_name}` at [..]/{script_name}
+    "});
+
+    let script_dir_path = temp_dir.path().join(SCRIPTS_DIR).join(script_name);
+
+    assert!(script_dir_path.exists());
+    assert!(script_dir_path.join("Scarb.toml").exists());
+    assert!(script_dir_path.join("src").exists());
+    assert!(script_dir_path.join("src/lib.cairo").exists());
+    assert!(script_dir_path
+        .join(format!("src/{script_name}.cairo"))
+        .exists());
+}
+
+#[test]
+fn test_script_init_files_content() {
+    let script_name = "my_script";
+    let temp_dir = TempDir::new().expect("Unable to create a temporary directory");
+
+    let snapbox = Command::new(cargo_bin!("sncast"))
+        .current_dir(temp_dir.path())
+        .args(["script", "init", script_name]);
+
+    snapbox.assert().stdout_matches(formatdoc! {r"
+        command: script init
+        status: Successfully initialized `{script_name}` at [..]/scripts/{script_name}
+    "});
+
+    let script_dir_path = temp_dir.path().join(SCRIPTS_DIR).join(script_name);
+    let scarb_toml_path = script_dir_path.join("Scarb.toml");
+
+    let scarb_toml_content = std::fs::read_to_string(&scarb_toml_path).unwrap();
+    let lib_cairo_content = std::fs::read_to_string(script_dir_path.join("src/lib.cairo")).unwrap();
+    let main_file_content =
+        std::fs::read_to_string(script_dir_path.join(format!("src/{script_name}.cairo"))).unwrap();
```

Comment:
This basically tests the same thing as previous test, right? so maybe we do not need the previous test at all?

Reply:
The initial idea was to have one test that checks files structure and another that checks files content. However if the structure is wrong, the second test will throw an error anyway, hence I am fine with removing the first test.